packages/cli/src/helpers.mts

```diff
@@ -10,6 +10,14 @@ import { has } from 'lodash-es';
 
 const DIR_NAME = getDirNameForCurrentFile(import.meta);
 
+// helpers when when working with url to generate service
+const DEFAULT_BRANCH = 'develop';
```

Comment:
There are times when we need to generate from feature branches and deploy to PR env. 
We should handle that use case too.

Possibly, take branch as input via env variable?

```suggestion
const DEFAULT_BRANCH = process.env.GITHUB_BRANCH || 'develop';
```

Reply:
done and tested this with a new branch and passing branch name in env variable